enginetest/queries/queries.go

```diff
@@ -9970,6 +9970,31 @@ from typestable`,
 			{"03.10.2003"},
 		},
 	},
+
+	{
+		Query: "select charset(null)",
+		Expected: []sql.Row{
+			{"binary"},
```

Comment:
weird that the docs say `charset()` returns `null` if the argument is `null`. I also see my version of MySQL returning `binary` here, so looks like MySQL's documentation is incorrect. 🤷🏻